src/picknik_dual_ur_arm_config/config/realsense_config_high_accuracy.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Do we need this file? Or can we use the same one from one of the base packages in this repo?

Reply:
Good question. Do you know where this file is used? I tried to find it in the studio repos but couldn't find any usage

Reply:
Yes, I found the file but not where it used and how I tell the code that is using this file to look in the picknik_ur_base_config install directory.

Something like: realsense_config_file:=/picknik_ur_base_config/config/realsense_config_high_accuracy.json

Reply:
yeah, my searches yield that it's set in the `config.yaml` as follows:

https://github.com/PickNikRobotics/moveit_studio_ur_ws/blob/main/src/picknik_ur_base_config/config/config.yaml#L61-L63

Reply:
Thanks!